packages/ui-metric/src/MetricGroup/__examples__/MetricGroup.examples.tsx

```diff
@@ -25,8 +25,7 @@
 import React from 'react'
 import { Metric } from '../../Metric'
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'textAlign' implicitly has an 'any' type... Remove this comment to see the full error message
-const getMetric = (textAlign) => {
+const getMetric = (textAlign: any) => {
```

Comment:
I'm wondering if it worth typing these... We know that e.g. here the value are `'start' | 'end' | 'center'`, we could even export this type from Metric to use here, but maybe we don't win anything by it.

Reply:
we could export `Metric`'s props here, but I'm inclined not to do it, because it would not be used in other places